app/views/occupation_standards/show.html.erb

```diff
@@ -77,22 +77,25 @@
-                                        View Original Document
-                                    </div>
-                                </div>
-                            <% end %>
+                          <%= link_to @occupation_standard.original_file_url do %>
+                            <div class="flex w-full items-center space-x-4 rounded bg-slate-700 p-2 text-left text-white hover:bg-slate-600">
+                              <span>
+                                <svg aria-label="pdf icon" xmlns="http://www.w3.org/2000/svg" width="49" height="48" fill="none">
+                                  <path stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="4" d="M8.5 22V8a4 4 0 0 1 4-4h16l12 12v24a4 4 0 0 1-4 4h-24a4 4 0 0 1-4-4v-2" />
+                                  <path stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="4" d="M28.5 4v12h12" />
+                                  <path fill="#fff" d="M4.927 27.213v3.168c0 .068.044.111.13.13.33.04.592.059.786.059.697 0 1.235-.16 1.613-.48.387-.329.581-.784.581-1.366 0-1.153-.731-1.73-2.194-1.73-.243 0-.51.025-.8.073-.077.02-.116.068-.116.146Zm-1.599 8.661a.795.795 0 0 1-.58-.247.795.795 0 0 1-.248-.58v-8.88c0-.233.078-.441.233-.625a.824.824 0 0 1 .58-.306 25.456 25.456 0 0 1 2.384-.116c3.1 0 4.65 1.153 4.65 3.459 0 1.25-.362 2.2-1.09 2.848-.716.64-1.772.96-3.167.96-.369 0-.712-.015-1.032-.044-.087 0-.131.039-.131.116v2.587a.795.795 0 0 1-.247.581.795.795 0 0 1-.581.247h-.77ZM14.237 27.271v6.598c0 .087.043.14.13.16.194.039.427.058.698.058.979 0 1.7-.286 2.166-.858.474-.571.712-1.506.712-2.804 0-1.163-.238-2.016-.712-2.558-.475-.543-1.197-.814-2.166-.814-.271 0-.504.02-.697.058-.088.02-.131.073-.131.16Zm-1.599 8.633a.845.845 0 0 1-.581-.291.963.963 0 0 1-.233-.64v-8.806c0-.243.078-.451.233-.625a.824.824 0 0 1 .581-.306 21.931 21.931 0 0 1 2.238-.116c1.735 0 3.072.46 4.011 1.38.94.921 1.41 2.23 1.41 3.925 0 1.84-.47 3.236-1.41 4.185-.94.94-2.276 1.41-4.01 1.41-.737 0-1.483-.039-2.239-.116ZM22.687 35.874a.795.795 0 0 1-.582-.247.796.796 0 0 1-.247-.58v-8.953c0-.223.083-.417.247-.582a.796.796 0 0 1 .582-.247h4.985c.223 0 .416.083.58.247a.796.796 0 0 1 .248.582v.334a.796.796 0 0 1-.247.581.795.795 0 0 1-.581.247H24.46c-.087 0-.131.04-.131.117v2.165c0 .087.044.13.13.13h2.922c.223 0 .417.083.581.248a.749.749 0 0 1 .247.567v.232a.82.82 0 0 1-.828.814H24.46c-.087 0-.131.044-.131.131v3.386a.795.795 0 0 1-.247.581.75.75 0 0 1-.567.247h-.828Z" />
+                                </svg>
+                              </span>
+
+                              <div class="font-bold">
+                                View Original Document
+                              </div>
+                            </div>
+                          <% end %>
+                        <% else %>
+                          <p class="text-white">No documents available at the moment</p>
                         <% end %>
+                        <!--
```

Comment:
Do we want to comment this out, or should this be `unless public_document` ...?

Reply:
@littleforest we are going to enable this once we have a way to redact documents. This will chance one we have that feature enabled